pages/dashboard/projects.md

```diff
@@ -0,0 +1,371 @@
+
+[//]: <> (TODO - Adicionar links - verificar todos os items da lista)
+
+1. Clique na guia **Execuções** de seu projeto no 
+[Test Runner](https://docs.cypress.io/guides/core-concepts/test-runner.html).
+2. Clique em **Conectar ao Dashboard**.
+3. Você precisará fazer login para gravar seus testes, portanto, será solicitado que você faça login 
+no Cypress Dashboard aqui, caso ainda não tenha feito isso.
+
+    ![Setup a project](https://docs.cypress.io/img/dashboard/projects/setup-a-project-1.2e5fcf2c.png)
+ 
+4. Escolha quem é o dono do projeto. Você pode ser o dono ou selecionar uma organização que criou. 
+Se você não tiver nenhuma organização, clique em **Criar Organização**. As organizações funcionam 
+exatamente como no GitHub. Eles permitem que você separe seus projetos pessoais e de trabalho. 
+[Leia mais sobre organizações](https://docs.cypress.io/guides/dashboard/organizations.html).
+
+    ![Setup a project](https://docs.cypress.io/img/dashboard/projects/setup-a-project-2.36fe8b32.png)
+
+5. Se você não tiver nenhum projeto existente, terá a oportunidade de criar um novo aqui. 
+Se você tem projetos existentes e deseja criar um novo, pode clicar em **Criar Novo Projeto** para fazer um novo.
+
+    - Preencha o nome do seu projeto (isso é apenas para fins de exibição e pode ser alterado posteriormente).
+    - Escolha se este projeto é Público ou Privado.
+      - **Um projeto público** pode ter suas gravações e execuções vistas por qualquer pessoa. 
+Normalmente, esses são projetos de código aberto.
+      - **Um projeto privado** restringe seu acesso 
+[apenas aos usuários que você convida](https://docs.cypress.io/guides/dashboard/users.html). 
+
+    ![Setup a project](https://docs.cypress.io/img/dashboard/projects/setup-a-project-3.1e1b3934.png)
+
+6. Como alternativa, se você já criou um projeto no Dashboard, você pode linkar seu projeto 
+selecionando no menu *dropdown*. Certifique-se de selecionar um projeto limpo que não tenha sido 
+vinculado anteriormente a um projeto existente.
+
+    ![Setup a project](https://docs.cypress.io/img/dashboard/projects/setup-a-project-4.66a081bd.png)
+
+7. Clique em **Configuração de Projeto**.
+8. Agora você deve ver uma visão explicando como registrar sua primeira execução com sua chave de registro. 
+
+    ![Setup a project](https://docs.cypress.io/img/dashboard/projects/setup-a-project-5.15b5679e.png)
+
+9. Depois de configurar seu projeto, o Cypress insere um 
+[projectId](###id-do-projeto) exclusivo em seu arquivo de configuração, 
+`cypress.json` por padrão. Se você estiver usando o controle de origem, recomendamos que você 
+verifique seu arquivo de configuração, incluindo o `projectId`, no controle de origem.
+
+10. Dentro da [integração contínua](https://docs.cypress.io/guides/continuous-integration/introduction.html), 
+ou a partir do terminal do seu computador local, passe a [Chave de Registro](###chave-de-registro) exibida 
```

Comment:
Same here, the link with ### is not working.

Reply:
It's working to me 🤔